index.html

```diff
@@ -22,26 +24,189 @@
+              <p class="bold">CANOPY</p>
+              <p class="middot">&bull;</p>
+              <p>Back End Dev</p>
+              <p class="middot">&bull;</p>
+              <p>2015</p>
+            </div>
+            <p class="project-description">A daily selection of privately personalized reads;
+               no accounts or sign-ups required.</p>
+            <ul class="buttons-flex">
+              <li class="li-button">html</li>
+              <li class="li-button">css</li>
+              <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div> <!-- .project -->
+
+        <div class="project">
+          <img src="./images/works-section/project4.png" alt="Fourth project image">
+          <div class="text-content">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="project-data">
+              <p class="bold">CANOPY</p>
+              <p class="middot">&bull;</p>
+              <p>Back End Dev</p>
+              <p class="middot">&bull;</p>
+              <p>2015</p>
+            </div>
+            <p class="project-description">A daily selection of privately personalized reads;
+               no accounts or sign-ups required.</p>
+            <ul class="buttons-flex">
+              <li class="li-button">html</li>
+              <li class="li-button">css</li>
+              <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See Project</button>
+          </div>
+        </div> <!-- .project -->
+
+      </section> <!-- .works -->
       
-      
+      <section class="about-myself">
+        <h2 class="about">About<br>Myself</h2>
+        <p class="introduction"> 
+            Hello, I'm a software developer! I can help you build a product,
+           feature or website Look through some of my work and experience! 
+           If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <a href="https://linkedin.com/in/OscarFM-dev" class="connect-text">LET'S CONNECT</a>
+        <ul class="social-media-icons">
+          <li>
+            <a href="https://linkedin.com/in/OscarFM-dev">
+              <img src="./images/header-section/linkedin.png" alt="linkedin logo" class="linkedin-logo">
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/OscarFM-dev">
+              <img src="./images/header-section/github.png" alt="github logo">
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/OscarFMdev">
+              <img src="./images/header-section/twitter.png" alt="twitter logo">
+            </a>
+          </li>
+          <li>
+            <a href="https://www.instagram.com/oscarfmdev/">
+              <img src="./images/header-section/peace.png" alt="peace logo">
+            </a>
+          </li>
+        </ul>
```

Comment:
- [ ] Please, we could complete the icons with Medium, there are five in the Figma template. ✋🏽 

![image](https://user-images.githubusercontent.com/58642949/185724116-d942d799-dad6-41f0-8a05-3e534b6dd29a.png)